encoding/int/generate.go

```diff
@@ -0,0 +1,3 @@
+package int
```

Comment:
Conventionally we put `generate.go` for mocks in `generated/mocks/`. Perhaps move it there?

For the Makefile, perhaps follow the convention of mock generation in `https://github.com/m3db/m3aggregator/blob/master/Makefile`? Try copying any lines containing `mock` and dependencies and run `make mock-gen` from the top level dir, and you should see the generated mock files in `encoding/int/`

Reply:
I'll move it and edit the `Makefile`.